hk_adc/hk_adc.h

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
GPL header. Never mind all the lines > 80 cols, in the interest of saving time, just leave them as they are!

Reply:
I made the changes :) It did not take much time

Reply:
Thank you, much appreciated -- and it looks much better now! 🙂